deploy/chart/templates/api-deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Chart.Name }}-api
+  labels:
+    app: {{ .Chart.Name }}-api
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: {{ .Chart.Name }}-api
+  template:
+    metadata:
+      labels:
+        app: {{ .Chart.Name }}-api
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}-api
+          image: {{ .Values.apiImage.name }}:{{ .Values.apiImage.tag }}
+          command: ["uvicorn", "rope.api.main:app", "--host", "0.0.0.0", "--port", "80", "--timeout-keep-alive", "120"]
+          imagePullPolicy: Always
+          env:
```

Comment:
@P-Gill97 , @MReyna12 - Just as FYI: These are the current environment variables being set for the API. I think I captured everything we currently reference in `settings.py`. I expect when we get to the course build jobs we'll have to add an SQS queue and also create a service account + IAM role with corresponding permissions, but I left that out for the time being :sweat_smile: .